src/large.c

```diff
@@ -88,8 +89,13 @@ large_ralloc_no_move_expand(tsdn_t *tsdn, edata_t *edata, size_t usize,
 	size_t new_size = usize + sz_large_pad;
 
 	szind_t szind = sz_size2index(usize);
+
+	bool deferred_work_generated;
 	bool err = pa_expand(tsdn, &arena->pa_shard, edata, old_size, new_size,
-	    szind, zero);
+	    szind, zero, &deferred_work_generated);
+
+	assert(deferred_work_generated == false);
```

Comment:
Same here as in the slab alloc case; I'm not sure this will be true once we have "real" realloc support in the HPA.

Reply:
Right. For some reason I thought I'd leave the assert here and after we have the support we'll change that. But really, why not implement it right now.